### ISSUE TITLE
Typescript error

### ISSUE THREAD

**Issue by promraungdet** [overextended/ox_lib#416]
i make fivem project with typescript but error when build script 

`node_modules/@overextended/ox_lib/client/resource/callback/index.ts:3:37 - error TS7019: Rest parameter 'args' implicitly has an 'any[]' type.

3 const activeEvents: Record<string, (...args) => void> = {};
                                      ~~~~~~~

node_modules/@overextended/ox_lib/client/resource/callback/index.ts:44:58 - error TS7019: Rest parameter 'args' implicitly has an 'any[]' type.

44 export function onServerCallback(eventName: string, cb: (...args) => any) {
                                                            ~~~~~~~

node_modules/@overextended/ox_lib/client/resource/callback/index.ts:45:71 - error TS7019: Rest parameter 'args' implicitly has an 'any[]' type.

45   onNet(`__ox_cb_${eventName}`, async (resource: string, key: string, ...args) => {
                                                                         ~~~~~~~

node_modules/@overextended/ox_lib/shared/resource/cache/index.ts:7:28 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ resource: string; }'.
  No index signature with a parameter of type 'string' was found on type '{ resource: string; }'.

7       const result = key ? target[key] : target;
                             ~~~~~~~~~~~

node_modules/@overextended/ox_lib/shared/resource/cache/index.ts:11:9 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ resource: string; }'.
  No index signature with a parameter of type 'string' was found on type '{ resource: string; }'.

11         target[key] = value;
           ~~~~~~~~~~~

node_modules/@overextended/ox_lib/shared/resource/cache/index.ts:14:7 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ resource: string; }'.
  No index signature with a parameter of type 'string' was found on type '{ resource: string; }'.

14       target[key] = exports.ox_lib.cache(key) || false;
         ~~~~~~~~~~~

node_modules/@overextended/ox_lib/shared/resource/cache/index.ts:15:14 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ resource: string; }'.
  No index signature with a parameter of type 'string' was found on type '{ resource: string; }'.

15       return target[key];
                ~~~~~~~~~~~


Found 7 errors in 2 files.`

**Comment by LukeWasTakenn** [comment]
This seems to be an issue with some bundlers but I really can't figure out the exact cause, using Project Error's typescript boilerplate it works fine.

What bundler are you using and what are your tsconfig settings?

**Comment by promraungdet** [comment]
you use project typescript from fivem-typescript-boilerplate this tsconfig [tsconfig.json](https://github.com/project-error/fivem-typescript-boilerplate/blob/master/server/tsconfig.json)

Now I solved the problem by adding type any to the errors. But I have a problem when I install a new package. The value I changed will come back as before. I would like you to update it by adding type any ?

**Comment by LukeWasTakenn** [comment]
Should be resolved with 61e3ba9
